README.rst

```diff
@@ -522,6 +522,14 @@ Choose from the list of available fixers:
                         Align equals symbols in
                         consecutive lines.
 
+* **array_element_space** [contrib]
+                        In array declaration, there
+                        MUST NOT be a space before
+                        each comma, there MUST be one
+                        space after each comma and
+                        there should be space around
```

Comment:
I went throughout all the `array` or `=>` related fixers and they changing white spaces around `=>` or aligning it for multi-line arrays. There is no need for another one. The only missing bit (if it is actually desirable) is to fix white-spaces around `=>` in single-line arrays.